README.md

```diff
@@ -41,4 +45,15 @@ Since ML Kit Vision is a Firebase library App has corresponding App in Firebase
 For security reasons `google-services.json` file is excluded from this repo.
 If you need access to that file get in touch via `Issues` tab.  
 Or create new project using instruction placed [here](https://codelabs.developers.google.com/codelabs/flutter-firebase/#6)  
-File `google-services.json` should be placed in Android `app` module.
```

Comment:
we'd need to document ios guide as well, info about placing google-services.plist in ios folder

Reply:
Yeah, I know but the thing is that I don't have macOS so I'm omitting iOS configuration since the beginning

Reply:
So I'd expect that iOS will have troubles compiling but don't have option to check this. On the other hand I'll rise an issue to represent iOS configuration/setup for all dependencies

Reply:
#9 for iOS config work
#10 for readme update with disclaimer that iOS needs attention